js/lib/ilib-web.js

```diff
@@ -58,11 +58,23 @@ var requireClass = function() {
 
     for (var i = 0; i < scripts.length; i++) {
         var source = scripts[i].src;
-        if (source && (pos = source.search(/ilib-web\.js$/)) !== -1) {
-            var url = new URL(source);
-            this.protocol = url.protocol + "//";
-            this.root = url.pathname;
+
+        if (navigator && navigator.userAgent && (navigator.userAgent.indexOf(" .NET") > -1) ){
+            // IE brower
```

Comment:
This should use ilib._getBrowser() instead of figuring it out on your own.

Reply:
I guess this part works before requiring ilib codes.

Reply:
Oh I see... so that means ilib._getBrowser() is not available yet. Okay